### PR TITLE
Update Core.php

### PR DESCRIPTION
**Pull request by gstesq**
Removed extra bracket

### CODE REVIEW

**Review by davidjconnelly — APPROVED** [review]
Okay cool!  I would have gone with:

(() || ()) {

It's a different syntax than what I'm use to but if it's cool then it's cool.  Cheers!

**Comment by gstesq** [comment]
Copied Jakes code suggestion. With same code copied in two places (see line 62), edit is to mirror first copy.

**Comment by jakecastelli** [comment]
I am sure it is cool 👍

File: engine/Core.php [hunk 1]
```diff
@@ -109,7 +109,7 @@ private function serve_child_module_asset($asset_path, $file_name) {
 
                     $content_type = mime_content_type($asset_path);
 
-                    if ($content_type == 'text/plain'|| $content_type == 'text/html')) {
+                    if ($content_type == 'text/plain'|| $content_type == 'text/html') {
```
Comment (delboy1978uk) [inline engine/Core.php:112]: when comparing strings you can *always* use `===` ;-)

**Pull request merged**